### PR TITLE
Kgashok/gitpod setup

### PR DESCRIPTION
**Pull request by kgashok**
a test  PR as requested today

### CODE REVIEW

**Comment by lakesare** [comment]
There are actually merge conflicts in the Readme, could you resolve them locally & push, @kgashok?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+
+# Contributing
+
```
Comment (lakesare) [inline CONTRIBUTING.md]: @kgashok, could you remove this file please?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
+# Added today
```
Comment (lakesare) [inline README.md]: 👍 for the link to Gitpod, but let's remove `# Added today` title.

**Comment by kgashok** [comment]
Removed CONTRIBUTING.md file

File: CONTRIBUTING.md (hunk 1 above)
Comment (kgashok) [reply CONTRIBUTING.md]: done!

File: README.md (hunk 2 above)
Comment (kgashok) [reply README.md]: Done

**Comment by lakesare** [comment]
Looks good!

**Pull request merged**